roles/openshift_node_group/files/sync.yaml

```diff
@@ -71,12 +64,49 @@ spec:
             touch /tmp/.old
           fi
 
+          # loop until BOOTSTRAP_CONFIG_NAME is set
```

Comment:
Ship this as part of some image?  This seems crufty to have a giant bash script embedded like this.

Reply:
Maybe later.  Config is config.  Image is not any easier to manage.

Reply:
In this case sync belongs to how we are configuring dynamic sync during install.  This is serverless bash

Reply:
Note that I will probably collapse a lot of `sdn.yaml` into the networking go binary once we get the transition locked down so it's stable.

In the future all of the sync code will be moving into the kubelet or another component, I just don't have that component now and I want to get the algorithm correct for now.